fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/NormalizeRepeat.java

```diff
@@ -116,6 +116,11 @@ private void checkGroupingSetsSize(LogicalRepeat<Plan> repeat) {
     }
 
     private LogicalAggregate<Plan> normalizeRepeat(LogicalRepeat<Plan> repeat) {
+        if (repeat.getGroupingSets().size() == 1 && ExpressionUtils
+                .collect(repeat.getOutputExpressions(), GroupingScalarFunction.class::isInstance).isEmpty()) {
+            return new LogicalAggregate<>(repeat.getGroupByExpressions(),
+                    repeat.getOutputExpressions(), Optional.empty(), repeat.child());
```

Comment:
we have three args ctor
```suggestion
            return new LogicalAggregate<>(repeat.getGroupByExpressions(),
                    repeat.getOutputExpressions(), repeat.child());
```